templates/web/base/admin/templates.html

```diff
@@ -0,0 +1,26 @@
+[% INCLUDE 'admin/header.html' title=tprintf(loc('Response Templates for %s'), body.name) -%]
```

Comment:
Not quite, because the zurich admin header doesn't put the `title` value in a <h1>.